administration/configuring-fluent-bit/yaml/environment-variables-section.md

```diff
@@ -2,6 +2,8 @@
 
 The `env` section lets you define environment variables directly within the configuration file. These variables can then be used to dynamically replace values throughout your configuration using the `${VARIABLE_NAME}` syntax.
 
+**Note:** Variables set in this section cannot be overridden by system environment variables.
```

Comment:
```suggestion
Variables set in this section cannot be overridden by system environment variables.
```

Our style guide doesn't offset notes like this. Most of the time, it's not needed. If it is, it should use a hint at the info level.